README.md

```diff
@@ -34,7 +34,7 @@ Within the reStructuredText files use the `sphinx_argparse_cli` directive that t
 | Name        | Description                                                                                                                   |
 | ----------- | ----------------------------------------------------------------------------------------------------------------------------- |
 | module      | the module path to where the parser is defined                                                                                |
-| func        | the module path to where the parser is defined                                                                                |
+| func        | the function name of the parser                                                                                               |
```

Comment:
the name of the function that once called with no arguments constructs the parser

Reply:
```suggestion
| func        | the name of the function that once called with no arguments constructs the parser                                             |
```

Right! Thanks, that makes sense.

Do you know of GitHub suggestions?
https://til.hashrocket.com/posts/szgdi4at59-github-insert-a-suggestion-

Reply:
Yes. I just don't like to use it 😆 too much hassle using it :-D